### PR TITLE
constants.DEFAULT_ATTRIBUTE_TYPE is changed to Text

### PR DESCRIPTION
**Pull request by Keshav-NEC**
Fix for issue #810

### CODE REVIEW

**Comment by fgalan** [comment]
Suggestion:

1. Create a PR at iotagent-node-lib fixing the functional test in that PR
2. In this PR (iota-json) modificy packages.json iotagent-node-lib dependency to pint to the fork repository and branch used in the PR in step 1

**Comment by AlvaroVega** [comment]
Maybe a twing PR for iotagent-ul applies ?
https://github.com/telefonicaid/iotagent-ul/blob/522990859c9e607c024bc85a81f23bbb3d9650f6/lib/constants.js#L40

**Comment by fgalan** [comment]
> Maybe a twing PR for iotagent-ul applies ? https://github.com/telefonicaid/iotagent-ul/blob/522990859c9e607c024bc85a81f23bbb3d9650f6/lib/constants.js#L40

I think so.

@Keshav-NEC could you do in iotagent-ul repository a twin PR as @AlvaroVega suggests, please?

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -0,0 +1 @@
+- Fix: DEFAULT_ATTRIBUTE_TYPE is changed to Text (#810)
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]:
```suggestion
- Fix: default attribute type changed from 'string' to 'Text' (#810)
```

That wording is "closer" to the user (DEFAULT_ATTRIBUTE_TYPE is for developers).

**Comment by Keshav-NEC** [comment]
> @Keshav-NEC could you do in iotagent-ul repository a twin PR as @AlvaroVega suggests, please?

Hi @fgalan, we are working on it.

File: package.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
     "body-parser-xml": "2.0.5",
     "dateformat": "3.0.3",
     "express": "4.18.1",
-    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "https://github.com/Keshav-NEC/iotagent-node-lib.git#Fix_for_1569",
```
Comment (fgalan) [inline package.json:71]: Now PR https://github.com/telefonicaid/iotagent-node-lib/pull/1571 has been merged, it's time to revert this change :)

```suggestion
    "iotagent-node-lib": "https://github.com/telefonicaid/iotagent-node-lib.git#master",
```

Comment (Keshav-NEC) [reply package.json:71]: Changes done as per comment received.

**Review by fgalan — APPROVED** [review]
LGTM. Thanks!
